cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraCluster.java

```diff
@@ -818,4 +829,28 @@ public ClusterJobStatus getLastClusterJob(ClusterJobType jobType) {
         }
         return null;
     }
+
+    public static void fillConfigRoleGaps(CassandraConfigRole.Builder configRole) {
```

Comment:
please move into `PersistedCassandraFrameworkConfiguration` when updating the default supplier.